frontend/talentsearch/src/js/components/search/SearchContainer.tsx

```diff
@@ -222,6 +220,22 @@ export const SearchContainer: React.FC<SearchContainerProps> = ({
 const SearchContainerApi: React.FC = () => {
   const [{ data, fetching, error }] = useGetSearchFormDataAcrossAllPoolsQuery();
   const skills = data?.skills;
+  const pools = data?.pools;
+
+  const availableClassifications = pools?.map(
+    (pool) => pool?.classifications && pool?.classifications[0],
+  );
```

Comment:
Huh, I would have thought the `pool?.classifications && pool?.classifications[0]` expression would result in a boolean, not a Classification, but the typescript says otherwise.😕 Anyway, I don't think it's a good idea here to assume that each pool only has one classification. I'd recommend replacing this with
```suggestion
  const availableClassifications = pools
    ?.flatMap((pool) => pool?.classifications)
    .filter(notEmpty);
```